_includes/head.html

```diff
@@ -1,5 +1,5 @@
 <!doctype html>
-<html class="no-js" lang="de">
+<html class="no-js" lang="{{ page.language }}">
```

Comment:
This is not yet set by `index.md` or `index.en.md`, is it? (Might be some magic that I didn't notice. At least it's not in the index.html built by Jekyll.)